modules/sdk-core/package.json

```diff
@@ -39,10 +39,12 @@
     "@bitgo/utxo-lib": "^2.4.1",
     "@noble/secp256k1": "1.6.0",
     "@stablelib/hex": "^1.0.0",
+    "bech32": "^2.0.0",
     "big.js": "^3.1.3",
     "bignumber.js": "^9.0.0",
     "bip32": "^3.0.1",
     "bitcoinjs-message": "^2.0.0",
+    "bolt11": "^1.4.0",
```

Comment:
Can these dependencies be moved to `sdk-coin-btc` without causing any problems?

Reply:
wouldn't it cause a circular dependency since `sdk-coin-btc` has `@bitgo/sdk-core` as a dependency and `validateLnurlInvoice.ts` lives in `@bitgo/sdk-core` ? I am not sure how to move it to `sdk-coin-btc`